src/icons/general/IconGlobe.js

```diff
@@ -12,18 +13,17 @@ type Props = {
     width?: number,
 };
 
-const IconGlobe = ({ className = '', color = '#000', height = 16, title, width = 16 }: Props) => (
+const IconGlobe = ({ className = '', color = bdlNeutral03, height = 16, title, width = 16 }: Props) => (
```

Comment:
32x32 please

Reply:
@priyajeet I've made it 20x20 as it's mainly used as a badgeable icon